### PR TITLE
Va70 pos doctype

### PR DESCRIPTION
**Pull request by Manjot-Vienna**
Work Completed for POS Document Type Payment Generation Automatically

### CODE REVIEW

File: ModelLibrary/Model/MOrder.cs [hunk 1]
```diff
@@ -3758,6 +3777,23 @@ public String CompleteIt()
                         String msg = invoice.GetProcessMsg();
                         if (msg != null && msg.Length > 0)
                             Info.Append(" (").Append(msg).Append(")");
+
+                        // for POS Doctype we need to create payment with invoice
+                        if (invoice != null)
```
Comment (Bharat-Vienna) [inline ModelLibrary/Model/MOrder.cs:3784]: Use it with Existing code, where same type of check is applied

File: ModelLibrary/Model/MOrder.cs [hunk 2]
```diff
@@ -2204,6 +2204,24 @@ protected override bool BeforeSave(bool newRecord)
                 if (newRecord && GetC_DocType_ID() == 0)
                     SetC_DocType_ID(0);
 
+                //in case of payment method is letter of credit and Check In POS Order Show message to change Payment Method
+                if (GetC_DocTypeTarget_ID() > 0)
```
Comment (Bharat-Vienna) [inline ModelLibrary/Model/MOrder.cs:2209]: Use it with Existing code, where same type of check is applied

File: ModelLibrary/Model/MOrder.cs [hunk 3]
```diff
@@ -2216,6 +2217,7 @@ protected override bool BeforeSave(bool newRecord)
                             if (paymentbaseType.Equals("L") || paymentbaseType.Equals("S"))
                             {
                                 _processMsg = Msg.GetMsg(GetCtx(), "VIS_PleaseChangePaymentMethod");
+                                log.SaveError("Error", Msg.GetMsg(GetCtx(), "VIS_PleaseChangePaymentMethod"));
```
Comment (Bharat-Vienna) [inline ModelLibrary/Model/MOrder.cs:2220]: remove Error from message

**Review by Amit-Vienna — APPROVED** [review]
fine

**Pull request merged**